### ISSUE TITLE
Big Crashe in android/Signing, MainActivity method updateUI

### ISSUE THREAD

**Issue by marcosmbbrito** [googlesamples/google-services#25]
Hello,

Because the code is not checking for: 
« if (Plus.PeopleApi.getCurrentPerson(mGoogleApiClient) != null) » 
source: https://developers.google.com/identity/sign-in/android/people 

The samples crashes with one of my account marcosmb.brito@gmail.com when it tries to getDisplayname().

The file with a bug is https://github.com/googlesamples/google-services/blob/master/android/signin/app/src/main/java/com/google/samples/quickstart/signin/MainActivity.java

Regards and continue with the nice job,

**Comment by samtstern** [comment]
@marcosmbbrito Thanks for the report.  This normally happens if you don't have the Google+ API enabled for your project in the Google Developers Console.  Try double-checking that and seeing if it fixes your issue.

I agree that such a null check would be useful, if you want to submit a pull request I could merge it.

**Comment by jbj88817** [comment]
I have the same issue, but it crashed in M preview 3, not on lollipop. Here is logcat:

Process: com.google.samples.quickstart.signin, PID: 13729
    java.lang.NullPointerException: Attempt to invoke interface method 'java.lang.String com.google.android.gms.plus.model.people.Person.getDisplayName()' on a null object reference
            at com.google.samples.quickstart.signin.MainActivity.updateUI(MainActivity.java:107)
            at com.google.samples.quickstart.signin.MainActivity.onConnected(MainActivity.java:173)
            at com.google.android.gms.common.internal.zzj.zzg(Unknown Source)
            at com.google.android.gms.common.api.zze.zzmH(Unknown Source)
            at com.google.android.gms.common.api.zze.onConnected(Unknown Source)
            at com.google.android.gms.common.api.zzg$2.onConnected(Unknown Source)
            at com.google.android.gms.common.internal.zzi$zzg.zznO(Unknown Source)
            at com.google.android.gms.common.internal.zzi$zza.zzc(Unknown Source)
            at com.google.android.gms.common.internal.zzi$zza.zzr(Unknown Source)
            at com.google.android.gms.common.internal.zzi$zzc.zznQ(Unknown Source)
            at com.google.android.gms.common.internal.zzi$zzb.handleMessage(Unknown Source)

**Comment by hnaoto** [comment]
I encountered exactly same issue and I have enabled Google+ API at the beginning. It turns out I didn't signed in the application properly.

Everything works well after following the instruction on this page- [App signing](https://developer.android.com/tools/publishing/app-signing.html)
It would be nice if the [Integrating Google Sign-In documentation](https://developers.google.com/identity/sign-in/android/start-integrating) get into signing in a little bit more  : ) 

Overall, thanks for the nice work  =D 

PS: I have tested on device (5.1.1) and emulator (API 23). Both works well.
